utest.h

```diff
@@ -263,6 +263,7 @@ static UTEST_INLINE utest_int64_t utest_ns(void) {
 #elif __EMSCRIPTEN__	                                    
 	return emscripten_performance_now()*1000000.0; 
 #endif
+  return 0;
```

Comment:
If you get here ever it'd mean the framework doesn't support your configuration.

The proper fix here would be to call the correct functions for MINGW (I guess the same as `__linux` perhaps?)

Reply:
Hmmm, I got this part working by calling the same functions as `_MSC_VER`, but then I'm getting utest main function redefinition errors. Then I solved those errors with two #ifdef's so to only define utest main once in my calling program. It feels a bit hacky so I I'll just keep that for my personal use and close the pull request.